pkg/pillar/cmd/zedmanager/updatestatus.go

```diff
@@ -371,10 +374,92 @@ func doUpdate(ctx *zedmanagerContext, uuidStr string,
 	return changed
 }
 
-func doInstall(ctx *zedmanagerContext, uuidStr string,
-	config types.AppInstanceConfig, status *types.AppInstanceStatus) (bool, bool) {
+// doInstallProcessStorageEntriesWithVerifiedImage
+//  Returns: (vs, storageStatusUpdate)
+func doInstallProcessStorageEntriesWithVerifiedImage(
```

Comment:
Its a bit odd that this function can modify AppInstanceStatus; I was expecting it to only modify StorageStatus.
That should at least be documented in this comment.

Also, there is no point in returning a VerifyImageStatus since the caller doesn't use that. 

Hence the function definition is a bit wonky.
checkVerifyImageStatus() (bool, bool) i.e., return (found, changed)
makes more sense to me.

Reply:
Thats a very observation. We actually don't see appInstanceStatus.ContainerId any more. Will remove the field. This also means I don;t need to pass AppInstanceStatus to the function. I just need the AppUUID.. Will fix it.